FhirToDataLake/deploy/templates/DeployAzureContainerApp.json

```diff
@@ -131,13 +160,6 @@
         "metadata": {
             "description": "Storage Account type"
         }
-    },
-    "location": {
-        "type": "string",
-        "defaultValue": "[resourceGroup().location]",
-        "metadata": {
-            "description": "Location for all resources."
-        }
     }
   },
   "variables": {
```

Comment:
also needs lower image reference for schema? to line 177

Reply:
Fixed, thanks.